src/legacy/views/preview/contact-preview-content.jsx

```diff
@@ -418,36 +427,129 @@ function ContactPreviewContent({ contact, onEdit, onDelete, onMail, onMove }) {
+							{contact.jobTitle && (
+								<ContactField field={contact.jobTitle} label={t('job.title', 'Job Role')} />
+							)}
+							{contact.department && (
+								<ContactField
+									field={contact.department}
+									label={t('job.department', 'Department')}
+								/>
+							)}
+							{contact.company && (
+								<ContactField field={contact.company} label={t('job.company', 'Company')} />
+							)}
 						</ContactPreviewRow>
-						<ContactPreviewRow></ContactPreviewRow>
+						{contact.notes && <ContactPreviewRow></ContactPreviewRow>}
+						{contact.notes && (
+							<ContactPreviewRow>
+								<ContactField
+									field={contact.notes}
+									label={t('label.notes', 'Notes')}
+									width="fill"
+								/>
+							</ContactPreviewRow>
+						)}
+						{addressData.length > 0 && (
+							<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+								<Text color="secondary" overflow="break-word" style={{ paddingLeft: '1rem' }}>
+									{t('section.title.address', 'Address')}
+								</Text>
+								{map(addressData, (address, index) => (
+									<ContactPreviewRow crossAlignment="flex-start" key={index}>
+										{address.type && (
+											<ContactField field={address.type} label={t('label.type', 'Type')} />
+										)}
+										{address.street && (
+											<ContactField
+												field={address.street}
+												label={t('section.field.street', 'Street')}
+											/>
+										)}
+										{address.city && (
+											<ContactField field={address.city} label={t('section.field.city', 'City')} />
+										)}
+										{address.postalCode && (
+											<ContactField
+												field={address.postalCode}
+												label={t('section.field.postalCode', 'PostalCode')}
+											/>
+										)}
+										{address.country && (
+											<ContactField
+												field={address.country}
+												label={t('section.field.country', 'Country')}
+											/>
+										)}
+										{address.state && (
+											<ContactField
+												field={address.state}
+												label={t('section.field.state', 'State')}
+											/>
+										)}
+									</ContactPreviewRow>
+								))}
+							</Row>
+						)}
 
-						<ContactPreviewRow>
-							<ContactField field={contact.notes} label={t('label.notes', 'Notes')} width="fill" />
-						</ContactPreviewRow>
+						{urlData.length > 0 && (
+							<Row width="fill" mainAlignment="flex-start" padding={{ top: 'large' }}>
+								<Text color="secondary" overflow="break-word" style={{ paddingLeft: '1rem' }}>
+									{t('label.website', 'Website')}
+								</Text>
+								{map(urlData, (url, index) => (
+									<ContactPreviewRow crossAlignment="flex-start" key={index}>
+										<Link
+											style={{ paddingLeft: '0.5rem' }}
+											underlined
+											onClick={() => {
+												openURLInNewTab(url.url);
```

Comment:
The opening in a new tab can be achieve using the `href` and `target="_blank"` props available on the `Link` component. This way we can avoid possible permission issues on opening a new tab using the `open` method